dev-tools/startup/sshuttle-server.sh

```diff
@@ -33,7 +33,7 @@ while sleep 5; do
 
     for addr in $alive; do
       fqdn="$(nslookup "$addr" | sed -n 's/.*name = \(.*\)/\1/p')"
-      hostname="$(sed -n 's/\([^\.]\+\).*/\1/p' <<< "$fqdn")"
+      hostname="$(sed -n 's/\([^\.]\+\.local\?\).*/\1/p' <<< "$fqdn")"
```

Comment:
@cheran-senthil this is to make hostnames like `node1.local` work. otherwise it was only adding `node1` and the fqdn (`node1.comms_default`) to /etc/hosts. which was a problem because it wouldn't resolve unless you added an extra period at the end in the browser (ex: only `node1./storage` and `node1.comms_default./storage` would work). the `.local` is an optional match, so it's fully compatible with the existing compose (I double-checked /etc/hosts)

Reply:
removed - no more need for .local as of now